apps/test-netzer-encoder/src/NetzerEncoder.cpp

```diff
@@ -0,0 +1,29 @@
+#include "NetzerEncoder.h"
+#include "mbed.h"
+
```

Comment:
one u add the namespace Encoder just do using namespace Encoder at the top of this file if youd like